src/fides/api/alembic/migrations/versions/xx_2025_12_18_1400_b9c8e7f6d5a4_add_monitor_steward_table.py

```diff
@@ -0,0 +1,69 @@
+"""add monitor steward table
+
+Revision ID: b9c8e7f6d5a4
+Revises: 2c067179b727
+Create Date: 2025-12-18 14:00:00.000000
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "b9c8e7f6d5a4"
+down_revision = "2c067179b727"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "monitorsteward",
+        sa.Column("id", sa.String(length=255), nullable=False),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column("user_id", sa.String(), nullable=False),
+        sa.Column("monitor_config_id", sa.String(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["monitor_config_id"], ["monitorconfig.id"], ondelete="CASCADE"
+        ),
+        sa.ForeignKeyConstraint(["user_id"], ["fidesuser.id"], ondelete="CASCADE"),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint(
+            "user_id", "monitor_config_id", name="uq_monitorsteward_user_monitor"
```

Comment:
decided to add this to enforce uniqueness, i.e. at the moment we'll only support a single link between users and monitors. i got worried about accidental duplicates and the complexity to support multiple types of joins was not worth it for this first increment - we only need a single join, and it shouldn't be hard to remove constraints as we go forward (the opposite is not true - harder to add constraints then remove them!)